AppCenter/AppCenter/Internals/Channel/MSOneCollectorChannelDelegatePrivate.h

```diff
@@ -67,4 +67,14 @@ extern NSString *const kMSLogNameRegex;
  * @return YES if data is valid, NO otherwise.
  */
 - (BOOL)validateLogData:(MSCSData *)data;
+
+/**
+ * Validate a dictionary that contains properties for a common schema log.
+ *
+ * @param properties The properties.
+ *
```

Comment:
In android we don't have blank lines between `@param` and `@return` ... Is this auto-formatted already?

Reply:
We don't have a convention around this, we sometimes have and sometimes don't have one. My preference would be to have `@param` and `@return` separated be an extra line.

Reply:
We do have a convention for this one @ElektrojungeMS and it follows the exact convention now. 😄